python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -259,16 +259,22 @@ def refresh_progress_bar(self, resource_manager: ResourceManager) -> None:
     def summary_str(self, resource_manager: ResourceManager) -> str:
         queued = self.num_queued() + self.op.internal_queue_size()
         active = self.op.num_active_tasks()
-        desc = f"- {self.op.name}: {active} active, {queued} queued"
+        desc = f"- {self.op.name}: Tasks: {active}"
         if (
             self.op._in_task_submission_backpressure
             or self.op._in_task_output_backpressure
         ):
-            desc += " 🚧"
-        desc += f", [{resource_manager.get_op_usage_str(self.op)}]"
+            desc += " [backpressured]"
+
+        # Add operator suffix, which includes actor information,
+        # directly after task details.
```

Comment:
Can we let the operators report `num_active/pending_actors` explicitly? 
Because other operators will use `progress_str` to report non-actor info, and it's weird to put here. 
for the actor pool map operator, the locality info can still be in `progress_str`

Reply:
i remember we had this discussion a while back, and we decided we didn't want to add operator-specific logic in this code area. should i add a check here, like `if isinstance(op, ActorPoolMapOperator)` explicitly and get the number of actors for `ActorPoolMapOperator`?

Reply:
We can add the method in PhysicalOperator, and default to return 0